Specs/DateTime/English/DateTimePeriodParser.json

```diff
@@ -1779,7 +1708,7 @@
   "Context": {
     "ReferenceDateTime": "2017-11-14T19:12:00"
   },
-  "NotSupportedByDesign": "python",
+  "NotSupported": "python",
```

Comment:
Any root reason it differs from JS?

Reply:
We can add a "Comment" attribute in the spec to document the issue. Similar to the open problems in some German datetime specs.

Reply:
This version is based on a previous JS version that does not support this test

Reply:
I see. Will bring python to parity with current JS come in the near future on the current plan?
We can merge this version first. Not a blocker.